src/Nuget/Sarif.Sdk.nuspec

```diff
@@ -30,6 +30,17 @@
     <file src='Sarif.dll' target='lib\net45\Sarif.dll'/>
     <file src='Sarif.pdb' target='lib\net45\Sarif.pdb'/>
     
-    <file src="..\..\..\..\src\Sarif\**\*.cs" target="src" />    
+    <file src="..\..\..\..\src\Sarif\**\*.cs" target="src" />
+
+    <file src="..\..\ConvertToSarif\AnyCPU_$configuration$\CommandLine.dll" target="tools\ConvertToSarif" />
+    <file src="..\..\ConvertToSarif\AnyCPU_$configuration$\ConvertToSarif.exe" target="tools\ConvertToSarif" />
+    <file src="..\..\ConvertToSarif\AnyCPU_$configuration$\ConvertToSarif.exe.config" target="tools\ConvertToSarif" />
+    <file src="..\..\ConvertToSarif\AnyCPU_$configuration$\ConvertToSarif.pdb" target="tools\ConvertToSarif" />
+    <file src="..\..\ConvertToSarif\AnyCPU_$configuration$\FSharp.Core.dll" target="tools\ConvertToSarif" />
```

Comment:
Just tried it. Doesn't seem to be required. Not sure why it's built into the output directory. Removed from .nuspec.

---
In reply to: [65741985](https://github.com/Microsoft/sarif-sdk/pull/320#discussion_r65741985) [](ancestors = 65741985)